paimon-web-api/src/main/java/org/apache/paimon/web/api/table/TableChange.java

```diff
@@ -0,0 +1,486 @@
+        }
+
+        @Override
+        public String toString() {
+            return "ModifyColumnName{"
+                    + "Column="
+                    + oldColumn
+                    + ", newName="
+                    + getNewColumnName()
+                    + '}';
+        }
+    }
+
+    /** A table change to modify the column comment. */
+    class ModifyColumnComment extends ModifyColumn {
+
+        private final String newComment;
+
+        private ModifyColumnComment(ColumnMetadata oldColumn, String newComment) {
+            super(oldColumn, oldColumn.withComment(newComment), null);
+            this.newComment = newComment;
+        }
+
+        /** Get the new comment for the column. */
+        public String getNewComment() {
+            return newComment;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            return (o instanceof ModifyColumnComment) && super.equals(o);
+        }
+
+        @Override
+        public String toString() {
+            return "ModifyColumnComment{"
+                    + "Column="
+                    + oldColumn
+                    + ", newComment='"
+                    + newComment
+                    + '\''
+                    + '}';
+        }
+    }
+
+    /** A table change to modify the column position. */
+    class ModifyColumnPosition extends ModifyColumn {
+
+        public ModifyColumnPosition(ColumnMetadata oldColumn, ColumnPosition newPosition) {
+            super(oldColumn, oldColumn, newPosition);
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            return (o instanceof ModifyColumnPosition) && super.equals(o);
+        }
+
+        @Override
+        public String toString() {
+            return "ModifyColumnPosition{"
+                    + "Column="
+                    + oldColumn
+                    + ", newPosition="
+                    + newPosition
+                    + '}';
+        }
+    }
+
+    /** A table change to drop the column. */
+    class DropColumn implements TableChange {
+
+        private final String columnName;
+
+        private DropColumn(String columnName) {
+            this.columnName = columnName;
+        }
+
+        /** Returns the column name. */
+        public String getColumnName() {
+            return columnName;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+            if (this == o) {
+                return true;
+            }
+            if (!(o instanceof DropColumn)) {
+                return false;
+            }
+            DropColumn that = (DropColumn) o;
+            return Objects.equals(columnName, that.columnName);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(columnName);
+        }
+
+        @Override
+        public String toString() {
+            return "DropColumn{" + "columnName='" + columnName + '\'' + '}';
```

Comment:
Is this implementation of `toString` wrong?

Reply:
I will update